releasenotes/notes/isolated-pipelines-be70e490c5450272.yaml

```diff
@@ -0,0 +1,15 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    Add `dogstatsd_pipeline_autoadjust_strategy` setting with `max_throughput` and `isolation`
+    strategies. `max_throughput` is the already existing strategy (when `dogstatsd_pipeline_autoadjust`
+    is `true`). `isolation` will let you create an arbitrary number of pipelines (based on `dogstatsd_pipeline_count`),
+    and will try to isolate containers based on their advertised origin. This will improve compression.
```

Comment:
This PR always uses `shardKeyGeneratorIsolation`, so this isolation happens regardless of the configured strategy?